internal/control_plane/command/root.go

```diff
@@ -18,34 +15,15 @@ var rootCmd = &cobra.Command{
 
 // Execute the root command and no error returned
 func Execute() {
+	rootCmd.AddCommand(start.NewCmd())
 	if err := rootCmd.Execute(); err != nil {
-		logger.Error("error in root command execution", err)
+		logger.Log.Err(err).Msg("Root command error")
+	} else {
+		logger.Log.Info().Msg("Root Command Executed")
 	}
 }
 
 func init() {
-	logger.Setup()
-	cobra.OnInitialize(initConfig)
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/config.json)")
 	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
```

Comment:
I guess we can remove the entire init function because it is not doing anything as such. Can you please check this?

Reply:
Yeah! for now there is no use, this was for purpose if config was in different location and we needed to use that config. Removed it.